src/main/java/org/folio/orders/events/handlers/ReceiptStatusConsistency.java

```diff
@@ -116,26 +117,28 @@ private void updateOrderStatus(PoLine poLine, Map<String, String> okapiHeaders,
     HelperUtils.sendEvent(MessageAddress.RECEIVE_ORDER_STATUS_UPDATE, messageContent, requestContext);
   }
 
-  private Future<ReceiptStatus> calculatePoLineReceiptStatus(PoLine poLine,
-      List<org.folio.rest.acq.model.Piece> pieces) {
+  private Future<ReceiptStatus> calculatePoLineReceiptStatus(PoLine poLine, List<Piece> pieces) {
```

Comment:
please extract 2 the same methods into one to avoid code duplication

Reply:
CheckinReceivePiecesHelper::calculatePoLineReceiptStatus is a bit different: it doesn't care about checkinItems flag, also it has yest another implementation of getPiecesQuantityByPoLineAndStatus.